### ISSUE TITLE
A11y: Support locallization for ariaLiveMessage

### ISSUE THREAD

**Issue by neet** [xyflow/xyflow#5201]
### Please describe the feature that you want to propose

Currently, React Flow offers a dynamic notification for assistive technology by Live Regions, but It is hard-coded in English and cannot be customized. We have been working on a project that aims to reach non-native English speakers, so supporting localization is essential.

Although it would be ideal if we could offer default messages in multiple languages that can be used without any configuration, selecting which i18n library is appropriate and prepare an architecture for it will take a time.

As a first step, it would be practical enough to start with supporting custom messages. We could do something like this:

```tsx
<ReactFlow
  nodes={nodes}
  edges={edges}
  i18n={{
    "a11yDescription.node.default": "Press enter or space to select a node. Press delete to remove it and escape to cancel.",
    "a11yDescription.node.keyboardDisabled": "Press enter or space to select a node. You can then use the arrow keys to move the node around. Press delete to remove it and escape to cancel.",
    "a11yDescription.edge.default": " Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.",
    "a11yDescription.live.moved": (node, x, y) => `Moved selected node ${node}. New position, x: ${x}, y: ${y}`,
  }}
/>
```

**Comment by moklick** [comment]
This would be a very nice improvement! We want to do a larger a11y update - for that we will probably include this one!

**Comment by moklick** [comment]
@printerscanner added a new prop called `ariaLabelConfig` in React Flow 12.7.0. This is the default:

```js
const defaultAriaLabelConfig = {
  'node.a11yDescription.default':
    'Press enter or space to select a node. Press delete to remove it and escape to cancel.',
  'node.a11yDescription.keyboardDisabled':
    'Press enter or space to select a node. You can then use the arrow keys to move the node around. Press delete to remove it and escape to cancel.',
  'node.a11yDescription.ariaLiveMessage': ({ direction, x, y }: { direction: string; x: number; y: number }) =>
    `Moved selected node ${direction}. New position, x: ${x}, y: ${y}`,
  'edge.a11yDescription.default':
    'Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.',

  // Control elements
  'controls.ariaLabel': 'Control Panel',
  'controls.zoomIn.ariaLabel': 'Zoom In',
  'controls.zoomOut.ariaLabel': 'Zoom Out',
  'controls.fitView.ariaLabel': 'Fit View',
  'controls.interactive.ariaLabel': 'Toggle Interactivity',

  // Mini map
  'minimap.ariaLabel': 'Mini Map',

  // Handle
  'handle.ariaLabel': 'Handle',
};
```
